

"""
fill in the fucntions so that the catch the appropriate errors and return "caught".

example:
>> test_list = []
>> catch_all_expeptions(test_list[10])
>> "caught"

"""


def catch_all_expeptions():
    pass


def catch_index_error():
    pass


def catch_value_error():
    pass



